### DISCUSSION TITLE
How can we attract more users to Zeus?

### DISCUSSION THREAD

**Discussion by aexol** [graphql-editor/graphql-zeus#291] [Q&A]
As @seanonthenet suggested I am adding this as a discussion topic. I started the process yesterday by adding 2 examples to documentation. I've also collected deep user feedback on what to add to docs.

**Comment by eliaharris** [comment]
@aexol  Zeus is a great tool, and I recommend it over other similar solutions. 

I think a big problem in regards to exposure is that SEO for similar search terms is dominated by `graphql-code-gen`

I also think the way Zeus handles mutation variables feels a bit redundant and could be off-putting to new adopters. I'm assuming that it's a limitation of Typescript's inference, but having to redeclare input types using `$(NAME, TYPE)` feels like extra work. The GraphQL endpoint defines what fields are required/nullable and the associated types. Ideally, the mutations would automatically infer the variables.

I'm planning on digging into the internals and seeing if I could implement something along those lines, but maybe you could explain/link to a previous PR that dives into why that design choice was made.

Thanks again for such an amazing tool!
